web-frontend/src/main.tsx

```diff
@@ -6,29 +6,31 @@ import {
   createRoutesFromElements,
   Route,
 } from 'react-router-dom';
-
 import './index.css';
 import '@massalabs/react-ui-kit/src/global.css';
 import Welcome from './pages/Welcome/Welcome.tsx';
 import SelectAccount from './pages/SelectAccount/SelectAccount.tsx';
 import Error from './pages/Error.tsx';
 import AddAccount from './pages/AddAccount/AddAccount.tsx';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+
+const queryClient = new QueryClient();
```

Comment:
This would be in the same specific MR to add the lib. I would be a small MR only concerning the addition.